src/components/buttons/flat-button/flat-button.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import { omit } from 'lodash';
-import vars from '../../../../materials/custom-properties';
+import Text from '@commercetools-uikit/text';
```

Comment:
Wondering if it would make sense to have the package named `@commercetools-uikit/typography`, which exports individual components like `import { Header, Body, ... }`